### ISSUE TITLE
When possible, glue should have a background explicitly assigned

### ISSUE THREAD

**Issue by jeremyfelt** [washingtonstateuniversity/WSU-spine#403]
This is not an immediate accessibility issue, but automated scanners have trouble seeing that the navigation text is on top of the background color assigned to `#glue:before`. In most cases, we should be able to apply this background color to glue itself, which will clear these automated errors.
